exercism/largest-series-product/largest-series-product.js

```diff
@@ -0,0 +1,26 @@
+const Series = function(series) {
+  if (!series.match(/^\d*$/g)) {
+    throw new Error('Invalid input.')
+  }
+  this.series = [...series]
```

Comment:
Also `[...series]` seems weird to me here. Why are you using the `...` notation?